examples/employment-based-loan/contracts/EmploymentLoan.sol

```diff
@@ -48,10 +45,9 @@ contract EmploymentLoan is SuperAppBase {
         AggregatorV3Interface _priceFeed,
         uint8 _priceFeedDecimals
     ) {
-        host = _host;
-        cfa = IConstantFlowAgreementV1(
+        IConstantFlowAgreementV1 cfa = IConstantFlowAgreementV1(
             address(
-                host.getAgreementClass(
+                _host.getAgreementClass(
                     keccak256(
```

Comment:
If CFA_ID is constant this should change

Reply:
@ngmachado @JoshuaTrujillo15 see most recent commit

CFA_ID has been added as a constant variable and the above syntax has been simplified